Makefile

```diff
@@ -137,6 +140,10 @@ gadget-%-container:
 push-gadget-%-container:
 	docker push $(CONTAINER_REPO):$(IMAGE_TAG)$(if $(findstring core,$*),-core,)
 
+gadget-%-container-debug:
+	docker buildx build -t $(CONTAINER_REPO):$(IMAGE_TAG)$(if $(findstring core,$*),-core,) --build-arg DEBUG=1 --build-arg DEBUG_PORT=$(DEBUG_PORT) -f Dockerfiles/gadget-$*.Dockerfile \
```

Comment:
In my opinion, the choice of `$DEBUG_PORT` should not be at build-time but at run-time. What about removing `--build-arg DEBUG_PORT` here and let the user add it in the yaml if they want to change the default from 4040 to something else?